config/samples/backends/lvm-iscsi/multipathd.yaml

```diff
@@ -0,0 +1,29 @@
+# Includes the /etc/multipathd.conf contents and the systemd unit changes
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: master
+  name: 99-master-custom-enable-multipathd
+spec:
+  config:
+    ignition:
+      version: 3.2.0
+    storage:
+      files:
+        - path: /etc/multipath.conf
+          overwrite: false
```

Comment:
I'm not sure how this is supposed to work. This MachineConfig seems to just define the multipath.conf file, but doesn't the service need to be started? Does it need to be installed?

Reply:
You can see how we start it on L26-L29 of this same file.
I mention it in the comment of the first line.